docs/faq.md

```diff
@@ -24,6 +24,26 @@ Linux wheels are built in the [`manylinux` docker images](https://github.com/pyp
 
 - Alternative dockers images can be specified with the `CIBW_MANYLINUX_X86_64_IMAGE`, `CIBW_MANYLINUX_I686_IMAGE`, and `CIBW_MANYLINUX_PYPY_X86_64_IMAGE` options to allow for a custom, preconfigured build environment for the Linux builds. See [options](options.md#manylinux-image) for more details.
 
+### Building non-native architectures using emulation  {: #emulation}
+
+cibuildwheel supports building non-native architectures on Linux, via
+emulation through the binfmt_misc kernel feature. The easiest way to use this
```

Comment:
To be pedantically correct: "emulation through QEMU and integration through the binfmt_misc kernel feature"

As far as I understood, `binfmt` is really just the feature that links an executable file to the right way of executing it, but QEMU is doing all the hard emulation work here, right?

Reply:
I think another emulator that integrated via `binfmt_misc` would work here too, though? (I don't know if such a thing exists, however!)

Reply:
Oh, that's true. Maybe if you look at it that way, then yes :-)